src/logger.js

```diff
@@ -161,6 +161,7 @@ module.exports = class Logger {
       }
     } else {
       this.fatal('Failed to compile subgraph', e)
+      process.exitCode = 1
```

Comment:
Let's leave it to the caller to decide whether failing to compile a subgraph is reason enough to exit with an error code. This doesn't belong in the logger.